src/libcollections/lib.rs

```diff
@@ -52,6 +52,7 @@
 #![feature(shared)]
 #![feature(slice_get_slice)]
 #![feature(slice_patterns)]
+#![feature(sort_unstable)]
```

Comment:
I think Travis is failing on this line during tests: https://travis-ci.org/rust-lang/rust/jobs/212204751